src/site/content/en/payments/android-payment-apps-delegation/index.md

```diff
@@ -0,0 +1,408 @@
+[AndroidManifest.xml](/android-payment-apps-overview/#androidmanifest.xml-2).
+
+```xml
+<activity
+  android:name=".PaymentActivity"
+  ...
+  <meta-data
+    android:name="org.chromium.payment_supported_delegations"
+    android:resource="@array/supported_delegations" />
+</activity>
+```
+
+The `resource` must be a list of strings chosen from the following valid values: 
+
+```json
+[ "payerName", "payerEmail", "payerPhone", "shippingAddress" ]
+```
+
+The following example can only provide a shipping address and the payer's email
+address.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string-array name="supported_delegations">
+    <item>payerEmail</item>
+    <item>shippingAddress</item>
+  </string-array>
+</resources>
+```
+
+## Parse `PAY` intent extras for required payment options
+
+The merchant can specify additional required information using the
+[`paymentOptions`](https://www.w3.org/TR/payment-request/#paymentoptions-dictionary)
+dictionary. Chrome will provide the list of required options that your app can
+provide by passing the following parameters to the `PAY` activity as [Intent
+extras](/android-payment-apps-overview/#parameters-2).
+
+### `paymentOptions`
+
+The subset of merchant specified payment options for which your app has declared
+delegation support.
+
+```kotlin
+val paymentOptions: Bundle? = extras.getBundle("paymentOptions")
+val requestPayerName: Boolean? = paymentOptions?.getBoolean("requestPayerName")
+val requestPayerPhone: Boolean? = paymentOptions?.getBoolean("requestPayerPhone")
+val requestPayerEmail: Boolean? = paymentOptions?.getBoolean("requestPayerEmail")
+val requestShipping: Boolean? = paymentOptions?.getBoolean("requestShipping")
+val shippingType: String? = paymentOptions.getString("shippingType")
+```
+
+It can include the following parameters:
+
+* `requestPayerName` - The boolean indicating whether or not the payer's name
+  is required.
+* `requestPayerPhone` - The boolean indicating whether or not the payer's phone
+  is required.
+* `requestPayerEmail` - The boolean indicating whether or not the payer's email
+  is required.
+* `requestShipping` - The boolean indicating whether or not shipping information
+  is required.
+* `shippingType` - The string showing the type of shipping. Shipping type can be
+  `"shipping"`, `"delivery"`, or `"pickup"`. Your app can use this hint in its
+  UI when asking for the user's address or choice of shipping options.
+
+### `shippingOptions` {: #shipping-options }
+
+The parcelable array of merchant specified shipping options. This parameter will
+only exist when `paymentOptions.requestShipping == true`.
+
+```kotlin
+val shippingOptions: List<ShippingOption>? =
+    extras.getParcelableArray("shippingOptions")?.mapNotNull {
+        p -> from(p as Bundle)
+    }
+```
+
+Each shipping option is a `Bundle` with the following keys.
+
+* `id`: The shipping option identifier.
```

Comment:
Please make the formatting of this consistent with the list that's provided in the paymentOptions section. This section uses `:` as a delimiter whereas that other section uses `-`